public/js/md-editor.js

```diff
@@ -36,3 +36,7 @@ const easyMDE = new EasyMDE({
     }
   }]
 })
+
+if (description != null) {
```

Comment:
OK, looks like I was wrong with this one. When we navigate to `groups/new` we don't pass any variables to the view, so `description` won't exist and we get an error saying `Uncaught ReferenceError: description is not defined`. In this case your initial check with `typeof description !== 'undefined'` works better. Sorry about that.

Reply:
No problem, important is we both learnt from that. 🤓